documentation/src/main/asciidoc/core/manual/en_US/02_getting_started.adoc

```diff
@@ -261,6 +261,11 @@ public class CriteriaBuilderFactoryProducer {
     private volatile CriteriaBuilderFactory criteriaBuilderFactory;
 
     public void init(@Observes @Initialized(ApplicationScoped.class) Object init) {
+        // no-op to force eager initialization
```

Comment:
Any reason why initalization in the observer is discouraged / deprecated now? Should I Change my code?

Reply:
Because of a possible initialization race between an EntityViewManagerProducer and CriteriaBuilderFactoryProducer. The current style could lead to a null cbf

Reply:
Thanks!